### PR TITLE
Field Partner Policy Updates

### PR DESCRIPTION
**Pull request by bryanjnelson**
┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/3629288915) by [Unito](https://www.unito.io)

### CODE REVIEW

File: src/components/authorization/policies/by-role/field-partner.policy.ts [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+  ),
+  r.EthnologueLanguage.when(member).read,
+  r.FieldRegion.read,
+  r.FieldZone.read,
+  r.FinancialReport.read,
+  r.Language.when(member).read,
+  r.Organization.when(member).read.specifically((p) => p.address.none),
+  r.Partner.when(member).read.specifically((p) => p.pointOfContact.none),
+  r.Partnership.when(member).read,
+  r.PeriodicReport.read,
```
Comment (CarsonF) [inline src/components/authorization/policies/by-role/field-partner.policy.ts:20]: Unless we are trying to also grant read access to `NarrativeReports` this isn't needed here as the two other concretes are specified. Or we could leave this and remove FinancialReport entry.

File: src/components/authorization/policies/by-role/field-partner.policy.ts [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
 
 @Policy(Role.FieldPartner, (r) => [
+  r.Ceremony.read,
+  inherit(
+    r.Engagement.read,
+    r.InternshipEngagement.when(member).edit.specifically(
+      (p) => p.ceremony.none
+    ),
+    r.LanguageEngagement.when(member).read
+  ),
```
Comment (CarsonF) [inline src/components/authorization/policies/by-role/field-partner.policy.ts:11]: They can read LanguageEngagement ceremonies but not Internship ones?

File: src/components/authorization/policies/by-role/field-partner.policy.ts [hunk 3]
```diff
@@ -1,6 +1,29 @@
...
+  r.FinancialReport.read,
+  r.Language.when(member).read,
+  r.Organization.when(member).read.specifically((p) => p.address.none),
+  r.Partner.when(member).read.specifically((p) => p.pointOfContact.none),
+  r.Partnership.when(member).read,
+  r.PeriodicReport.read,
+  r.Product.read,
+  r.Project.when(member).read.specifically((p) => p.rootDirectory.none),
+  r.ProjectMember.when(member).read,
+  r.StepProgress.read,
```
Comment (CarsonF) [inline src/components/authorization/policies/by-role/field-partner.policy.ts:24]: I doubt this is actually intentional compared to the StepProgress entry below.
```suggestion
```

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: src/components/authorization/policies/by-role/field-partner.policy.ts (hunk 1 above)
Comment (bryanjnelson) [reply src/components/authorization/policies/by-role/field-partner.policy.ts:20]: Nice catch...removing Financial Report.

File: src/components/authorization/policies/by-role/field-partner.policy.ts (hunk 3 above)
Comment (bryanjnelson) [reply src/components/authorization/policies/by-role/field-partner.policy.ts:24]: Right...cut/paste from the other policy.

File: src/components/authorization/policies/by-role/field-partner.policy.ts (hunk 2 above)
Comment (bryanjnelson) [reply src/components/authorization/policies/by-role/field-partner.policy.ts:11]: Should be both; a miss.  👍🏼

Comment (bryanjnelson) [reply src/components/authorization/policies/by-role/field-partner.policy.ts]: Talked to business...both should be read only.

**Review by CarsonF — APPROVED** [review]
🚀

**Pull request merged**